c7n/resources/rdscluster.py

```diff
@@ -658,3 +658,81 @@ def process(self, resources, event=None):
             if expected_dates.issubset(snapshot_dates):
                 results.append(r)
         return results
+
+
+@RDSCluster.filter_registry.register('db-cluster-parameter')
+class ParameterFilter(ValueFilter):
```

Comment:
can we subclass the rds ParameterFilter? looks like much of the recast logic here is copy pasted from there

Reply:
We can add a method like `get_parameter_groups` to make this generic across the 2 resource types

Reply:
Modified ParameterFilter in rds.py and used it as subclass "ClusterParameterFilter" in rdscluster.py. Please review and let me know if any further modification is needed